toolkit/scripts/mariner-required-configs.json

```diff
@@ -1114,6 +1114,22 @@
                 "PR": [
                     "https://github.com/torvalds/linux/commit/b561275d633bcd8e0e8055ab86f1a13df75a0269"
                 ]
+            },
+            "CONFIG_STX104": {
+                "value": [
+                    "is not set",
```

Comment:
for my understanding, why are we allowing pretty much any value for this one in particular?

Reply:
The config option entry is needed in order to allow the check to pass but we don't have a hard dependency on what the value is.

Reply:
To be fair, this probably could be fixed n the script to account for configs which only move like this.